src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -52,17 +52,6 @@ const TableHeader: React.FC<TableHeaderProps> = ({
     ? tables.find((table) => table.reference === seatedAtTable)
     : undefined;
 
-  const [error, setError] = useState<string>("");
-
-  const usersAtCurrentTable = useMemo(
-    () =>
-      seatedAtTable &&
-      recentVenueUsers.filter(
-        (user: User) => user.data?.[venueName]?.table === seatedAtTable
-      ),
-    [seatedAtTable, recentVenueUsers, venueName]
-  );
```

Comment:
Is this not needed anymore? And if not, why?

Reply:
This was used to calculate `max users - current users` now it just shows the total amount of seats

Reply:
Cool, makes sense.